addons/stock/__init__.py

```diff
@@ -4,4 +4,15 @@
 import controllers
 import models
 import report
-import wizard
\ No newline at end of file
+import wizard
+
+
+from odoo import api, SUPERUSER_ID
+
+
+# TODO: Apply proper fix & remove in master
+def pre_init_hook(cr):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env['ir.model.data'].search([
+        ('model', 'like', '%stock%')
```

Comment:
perhaps just `'stock%'`...

Reply:
can models have a prefix? If not, stock% is enough.

Reply:
there can indeed be models such as `report.stock.xml`, the thing is that these models may not be introduced by `stock` itself, in which case this would be overreaching.

Perhaps an extra domain `('module', '=', 'stock')` would be safer...